### PR TITLE
Feat: Pixiu connect with SpringCloud Server by nacos 

### PR DESCRIPTION
**Pull request by PhilYue**
**What this PR does**: 

Pixiu connect with SpringCloud Server

**Which issue(s) this PR fixes**:

Fixes # 
nope

**Special notes for your reviewer**:
nope

**Does this PR introduce a user-facing change?**:
nope

```release-note

```

### CODE REVIEW

File: pkg/registry/zookeeper.go [hunk 1]
```diff
@@ -18,6 +18,7 @@
 package registry
 
 import (
+	"github.com/apache/dubbo-go-pixiu/pkg/model"
```
Comment (AlexStocks) [inline pkg/registry/zookeeper.go:21]: move it to the 3rd import block.

File: pkg/adapter/springcloud/servicediscovery/servicediscovery.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+	ServiceEventListener interface {
+		OnAddServiceInstance(r *ServiceInstance)
+		OnDeleteServiceInstance(r *ServiceInstance)
+		OnUpdateServiceInstance(r *ServiceInstance)
+		GetServiceNames() []string
+	}
+
+	ServiceDiscovery interface {
+		// 直接向远程注册中心查询所有服务实例
```
Comment (AlexStocks) [inline pkg/adapter/springcloud/servicediscovery/servicediscovery.go:51]: translate it into english

File: pkg/adapter/springcloud/cloud.go [hunk 3]
```diff
@@ -64,58 +80,259 @@ func (p *CloudPlugin) Kind() string {
...
+	a.mutex.Lock()
+	defer a.mutex.Unlock()
+	cm.SetEndpoint(endpoint.Name, endpoint)
+}
+
+func (a *CloudAdapter) GetServiceNames() []string {
+	a.mutex.Lock()
+	defer a.mutex.Unlock()
+
+	var res []string
```
Comment (AlexStocks) [inline pkg/adapter/springcloud/cloud.go]: res := make([]string, 0, len(a.currentService))

File: pkg/registry/zookeeper.go (hunk 1 above)
Comment (PhilYue) [reply pkg/registry/zookeeper.go:21]: done

File: pkg/adapter/springcloud/servicediscovery/servicediscovery.go (hunk 2 above)
Comment (PhilYue) [reply pkg/adapter/springcloud/servicediscovery/servicediscovery.go:51]: done

File: pkg/adapter/springcloud/cloud.go (hunk 3 above)
Comment (PhilYue) [reply pkg/adapter/springcloud/cloud.go]: done

**Review by ztelur — APPROVED** [review]

**Review by xiaoliu10 — APPROVED** [review]

**Pull request merged**